strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -154,6 +173,22 @@ def loss(self, T, mode):
     def thermal_loss(self, T, nbar, mode):
         self.circuit.thermal_loss(T, nbar, mode)
 
+    def measure_fock(self, modes, shots=1, select=None):
+        if select is not None:
+            raise NotImplemented("Postselection is currently not supported "
+                                 "with measure_fock on the gaussian backend")
+
+        # TODO: add logic to make sure the correct modes are measured
```

Comment:
@josh146 @smite @nquesada Note the TODO here. Current implementation of `measure_fock` returns samples from all modes